code/include/swoc/Lexicon.h

```diff
@@ -813,4 +839,33 @@ bwformat(BufferWriter &w, bwf::Spec const &spec, Lexicon<E> const &lex) {
   return w;
 }
 
-}} // namespace swoc::SWOC_VERSION_NS
+}} // namespace swoc
+
+namespace std {
+
+template <size_t IDX, typename E> class tuple_element<IDX, swoc::detail::lexicon_pair_type<E>> {
+  static_assert("swoc::Lexicon::Pair tuple index out of range");
+};
+
+template <typename E> class tuple_element<0, swoc::detail::lexicon_pair_type<E>> {
+public:
+  using type = E;
+};
+
+template <typename E> class tuple_element<1, swoc::detail::lexicon_pair_type<E>> {
+public:
+  using type = swoc::TextView;
+};
+
+template <size_t IDX, typename E>
+auto
+get(swoc::detail::lexicon_pair_type<E> const &p) -> typename std::tuple_element<IDX, swoc::detail::lexicon_pair_type<E>>::type {
```

Comment:
Clang-Tidy: Modification of 'std' namespace can result in undefined behavior

Reply:
clang-tidy is just going to have to live with that.